src/pages/RFOX/components/Bridge/Bridge.tsx

```diff
@@ -0,0 +1,95 @@
+import { foxAssetId, foxOnArbitrumOneAssetId, fromAssetId } from '@shapeshiftoss/caip'
+import { AnimatePresence } from 'framer-motion'
+import { lazy, Suspense, useCallback, useState } from 'react'
+import { MemoryRouter, Route, Switch, useLocation } from 'react-router'
+import { makeSuspenseful } from 'utils/makeSuspenseful'
+import { selectFirstAccountIdByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { RfoxBridgeQuote } from './types'
+import { BridgeRoutePaths, type BridgeRouteProps } from './types'
+
+const suspenseFallback = <div>Loading...</div>
+
+const BridgeConfirm = makeSuspenseful(
+  lazy(() =>
+    import('./BridgeConfirm').then(({ BridgeConfirm }) => ({
+      default: BridgeConfirm,
+    })),
+  ),
+)
+
+const BridgeStatus = makeSuspenseful(
+  lazy(() =>
+    import('./BridgeStatus').then(({ BridgeStatus }) => ({
+      default: BridgeStatus,
+    })),
+  ),
+)
+
+const BridgeEntries = [BridgeRoutePaths.Confirm, BridgeRoutePaths.Status]
+
+export const Bridge: React.FC<BridgeRouteProps> = ({ headerComponent }) => {
+  return (
+    <MemoryRouter initialEntries={BridgeEntries} initialIndex={0}>
+      <BridgeRoutes headerComponent={headerComponent} />
+    </MemoryRouter>
+  )
+}
+
+export const BridgeRoutes: React.FC<BridgeRouteProps> = ({ headerComponent }) => {
+  const location = useLocation()
+
+  // TODO(gomes): remove dummy quote, and actually setBridgeQuote once we consume this as part of the staking flow -
+  // or maybe not even and just capture in closure, as we will just instantiate this component with the bridge quote already
+  // i.e when pushing to bridge, we will push with query args and use react-router to consume these here
```

Comment:
To be tackled in p2, this PR brings the whole *bridge* flow e2e, not the staking -> bridge flow